packages/office-ui-fabric-react/src/components/KeytipData/useKeytipData.ts

```diff
@@ -15,12 +15,15 @@ export interface IKeytipData {
  */
 export function useKeytipData(options: KeytipDataOptions): IKeytipData {
   const uniqueId = React.useRef<string>();
-  const keytipProps: IKeytipProps | undefined = options.keytipProps
-    ? {
-        disabled: options.disabled,
-        ...options.keytipProps,
-      }
-    : undefined;
+  // TODO: verify with @xugao whether this should ever change
```

Comment:
@xugao I changed this because the rule pointed out that the `useEffect` to register the keytip was missing dependencies. Do you think this is the right approach, or should I just disable the rule in this case?

Reply:
There's a test that's failing because of this change, so I think the way it was before was the intended behavior.

Reply:
Hmm interesting. I'm not sure the previous way was entirely correct either--if there's a different keytip, shouldn't it be re-registered?

Reply:
So here was the original logic before the conversion to FC/hook:
https://github.com/microsoft/fluentui/pull/13688/files#diff-ea9c0d7c983c01ae447822e6dd74bc0d

That's why i didn't have any deps here. 
I am a bit more lean towards disabling the rule for now. reason is that in existing user code, `keytipProps` can be passed as inlined prop (which busts memoization). I am nervous that this causes unintended side-effects if we add the deps

Reply:
Sounds good. Looking at the update logic, I do wonder if that will need to be handled a bit differently due to something @layershifter said in https://github.com/microsoft/fluentui/pull/14107#discussion_r457259549--I'm guessing the issue he mentioned has to do with concurrent mode but waiting for clarification. But if a fix is needed there, it can be handled in a separate PR.